apps/pie-microsite/src/_11ty/shortcodes/list.js

```diff
@@ -0,0 +1,20 @@
+const markdownFilter = require('../filters/markdown');
+
+/**
+ * A List HTML component
+ * @param {string} type - Type of list: orderd, pill
+ * @param {string} items - An array of list items
```

Comment:
I think the type for `items` should be `string[]` as it's an array of strings. Also I notice we use capitalised types in some JSDoc comments and not others. Probably worth us seeing which is valid and keeping it consistent